### DISCUSSION TITLE
Why getSubscriptions doesn't return basic plans from google play?

### DISCUSSION THREAD

**Discussion by victorroatt** [hyochan/react-native-iap#1694] [Q&A] [answered]
Hello guys, I'm trying to retrieve from the google play console the basic plans that I made it for my subscription. When I call the method getSubscriptions, I'm just receiving one of the three basic plans that I have it.

**Comment by luchoPenafiel** [comment]
Hey @victorroatt could you solve this? I'm facing with the same problem.

**Reply by victorroatt** [reply]
I solved by creating the subscriptions separately. This library is using the old API of IAP from google. Let's say you want to create a subscription plan and offer options like 1 month, 3 months and 1 year. If you follow the current instruction from google console you're going to have 1 subscription with 3 offers. But this module it's working in the older api where you have to create 3 subscriptions with 1 offer for each.

**Comment by andresesfm** [comment, marked as answer]
It's because this requires Google Play Billing v5. We are working on the migration here: https://github.com/dooboolab/react-native-iap/issues/1710. I have published [beta12](https://www.npmjs.com/package/react-native-iap/v/9.0.0-beta12). We could use some help testing it and sharing your feedback

**Reply by andresesfm** [reply]
v9.0.0 has been released

**Reply by skjangid** [reply]
Why getSubscriptions doesn't return prepaid (one time) plans from google play?

**Reply by andresesfm** [reply]
@skjangid did you try using `getProducts` instead?
